config/style_guides/ruby.yml

```diff
@@ -1061,3 +1061,4 @@ Rails/Delegate:
 Rails/HttpPositionalArguments:
   StyleGuide: http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/HttpPositionalArguments
   Enabled: false
+  
```

Comment:
I'm confused by this because there is a new line haha can you see whats going on locally for you? Otherwise 👍

Reply:
Hmm that is weird. I just have a new line at the end just like the master version.
![Screen Shot 2019-12-16 at 2 07 36 PM](https://user-images.githubusercontent.com/10731397/70947243-9d10a980-200d-11ea-930a-5eaa36f9bd70.png)

Reply:
Should be good to go now.